lib/ret_web/controllers/api-internal/v1/change_login_email_controller.ex

```diff
@@ -0,0 +1,37 @@
+defmodule RetWeb.ApiInternal.V1.ChangeLoginEmailController do
+  use RetWeb, :controller
+
+  alias Ret.Login
+
+  def post(conn, %{"old_email" => old_email, "new_email" => new_email})
+  when is_binary(old_email) and is_binary(new_email) do
+    conn = put_resp_header(conn, "content-type", "application/json")
```

Comment:
You probably want `json/2` rather than `put_resp_header/3`+`send_resp/3`+`Poison.encode/1`.

On a tangential note, prefer Jason to Poison.